### DISCUSSION TITLE
Help us name the 1.8 Release

### DISCUSSION THREAD

**Discussion by PedramNavid** [dagster-io/dagster#23030] [Community]
Hey folks! Pedram from the Dagster team here.

We'll be releasing Dagster v1.8 soon and want your feedback!

Every major and minor release gets named after a hit single or an album from a pop diva. Sometimes, the name of the song/album is inspired by what changes are coming with the release.

For inspiration, 1.8 will have:

A New Declarative Automation API
Data Catalog Improvements
New Asset Checks
Pipes are no longer experimental

And here are some songs/albums we've previously named our versions after!

[How Will I Know?](https://www.youtube.com/watch?v=m3-hY-hlhBg) (Whitney Houston)
[Material Girl](https://www.youtube.com/watch?v=xeddEMuM0dA) (Madonna)
[Smooth Operator](https://www.youtube.com/watch?v=4TYv2PhG89A) (Sade)
[Formation](https://www.youtube.com/watch?v=WDZJPJV__bQ) (Beyonce)
[Thank U, Next](https://www.youtube.com/watch?v=gl1aHhXnN1k) (Ariana Grande)
[Hello](https://www.youtube.com/watch?v=YQHsXMglC9A) (Adele)
[Cool for the Summer](https://www.youtube.com/watch?v=il9nqWw9W3Y) (Demi Lovato)
[Lucky Star](https://www.youtube.com/watch?v=ThHz9wlBeLU) (Madonna)
[The Edge of Glory](https://www.youtube.com/watch?v=QeWBS0JBNzQ) (Lady Gaga)
[Laundry Service](https://www.youtube.com/watch?v=7BPE5sR6oUU&list=PL035E8452800ACEA5) (Shakira)
[In the Zone](https://www.youtube.com/watch?v=we8h_2RFAyA) (Britney Spears)
[Waiting to Exhale](https://www.youtube.com/watch?v=_FT7ciAPuJk&list=PL_sN_rrP4od0gZ4C3LGkgcaRpVa_koQX4) (Whitney Houston, et al.)
[Love Plus One](https://www.youtube.com/watch?v=5_msHpEa3_Y) (Haircut 100)

[Listen to the full playlist](https://open.spotify.com/playlist/476UFttwzU8476ki1Aqcdd)

Drop a comment with your suggestions, and upvote the suggestions you like!

**Comment by PedramNavid** [comment]
[Beyonce - Upgrade U](https://www.youtube.com/watch?v=6nr8hPnZfMU)

**Comment by elmira-ahmedova** [comment]
[A Head Full of Dreams - Coldplay](https://www.youtube.com/watch?v=6YypytpsftA)

**Comment by MattyKuzyk** [comment]
A new callable API? Well then it has to be:

[Call Me Maybe](https://youtu.be/fWNaR-rxAic?si=4SeiAPqjlgslUfrA) (Carly Rae Jepsen)

**Comment by abi-mutinex** [comment]
Dagster does it again, so...

[Baby One More Time](https://www.youtube.com/watch?v=C-u5WLJ9Yk4&themeRefresh=1) (Britney Spears)

**Comment by maptagsde** [comment]
[Believe](https://www.youtube.com/watch?v=nZXRV4MezEw) (Cher)

**Comment by rlokugamage** [comment]
[Loud Pipes (Ratatat)](https://open.spotify.com/track/3qkFIjYRInFasy2jeDZPgm?si=fc69bff05a59422f)

**Comment by idor980** [comment]
[Mamma Mia (ABBA)](https://www.youtube.com/watch?v=unfzfe8f9NI)

**Comment by PedramNavid** [comment]
Call Me Maybe it is!

**Comment by machielg** [comment]
Tiny Dancer (Elton John)
